src/components/home/Balance.tsx

```diff
@@ -1,33 +1,48 @@
+import { previewerAddress } from "@exactly/common/generated/chain";
 import { ChevronDown } from "@tamagui/lucide-icons";
+import { Skeleton } from "moti/skeleton";
 import React from "react";
+import { Appearance } from "react-native";
 import { ms } from "react-native-size-matters";
 import { Accordion, Square, View } from "tamagui";
+import { zeroAddress } from "viem";
+import { useAccount } from "wagmi";
 
 import AssetList from "./AssetList";
+import { useReadPreviewerExactly } from "../../generated/contracts";
+import assetLogos from "../../utils/assetLogos";
+import AssetLogo from "../shared/AssetLogo";
 import Text from "../shared/Text";
 
 export default function Balance({ usdBalance }: { usdBalance: bigint }) {
+  const { address } = useAccount();
+  const { data: markets } = useReadPreviewerExactly({
+    address: previewerAddress,
+    account: address,
+    args: [address ?? zeroAddress],
+  });
+  const symbols = markets
+    ?.map(({ symbol, floatingDepositAssets }) => ({
+      floatingDepositAssets,
+      symbol: symbol.slice(3) === "WETH" ? "ETH" : symbol.slice(3),
+    }))
+    .filter(({ floatingDepositAssets }) => floatingDepositAssets > 0)
+    .map(({ symbol }) => symbol);
+
   return (
-    <View display="flex" justifyContent="center" backgroundColor="$backgroundSoft" gap="$s4">
-      <View display="flex" flexDirection="row" justifyContent="space-between" alignItems="center">
-        <Text
-          fontSize={ms(15)}
-          lineHeight={ms(21)}
-          fontWeight="bold"
-          color="$uiNeutralSecondary"
-          textAlign="center"
-          width="100%"
-        >
-          Assets
-        </Text>
-      </View>
-      <Accordion
-        overflow="hidden"
-        type="multiple"
-        borderRadius="$r3"
-        padding="$s4"
-        defaultValue={usdBalance === 0n ? undefined : ["balance"]}
-      >
+    <View
+      display="flex"
+      justifyContent="center"
+      borderWidth={1}
+      borderColor="$borderNeutralSoft"
+      backgroundColor="$backgroundSoft"
+      borderRadius="$r3"
+      paddingTop="$s3_5"
+      paddingBottom="$s3_5"
+      paddingRight="$s4_5"
```

Comment:
instead of setting both `paddingRight` and `paddingLeft` you can set only `paddingHorizontal`